api/lib/application/authentication/oidc/oidc-controller.js

```diff
@@ -12,4 +15,28 @@ module.exports = {
 
     return h.response({ redirectLogoutUrl }).code(200);
   },
+
+  async findUserForOidcReconciliation(request) {
+    const authenticationKey = request.query['authentication-key'];
+    const { email, password, identityProviderCode } = accountReconciliationSerializer.deserialize(request.payload);
```

Comment:
```suggestion
    const { email, password, identityProvider, authenticationKey } = request.deserializedPayload;
```

- On peut utiliser le rajout du `deserializedPayload`
- Comme préciser au-dessus la notion de code n'existe pas dans l'api, elle ne gère que le code donc reconnu comme identity provider

Reply:
ah mais oui, trop bien le `deserializedPayload`